src/main/java/de/uni/bremen/monty/moco/ast/declaration/FunctionDeclaration.java

```diff
@@ -254,64 +234,46 @@ public void visitChildren(BaseVisitor visitor) {
 		for (VariableDeclaration variableDeclaration : parameters) {
 			visitor.visitDoubleDispatched(variableDeclaration);
 		}
+//		if(wrapperFunctionObjectDeclaration != null){
```

Comment:
Unnecessary code is unnecessary (happens a few times)? Yet again, I don't really care.